building/trusted_applications.rst

```diff
@@ -373,35 +373,50 @@ third step the binary and its signature are stitched together into the full TA.
 Offline Signing of TAs
 ======================
 
-The TA dev kit does sign an application as last step of the linking process. For
-example, the file ``ta/arch/arm/link.mk`` in the :ref:`optee_os` source tree
-contains the statement
+There are two types of TAs that can be signed offline: the in-tree TAs, which come
+with the OP-TEE OS (for example the ``pkcs11`` TA) and any external TA coming from the user. Offline signing
+runs directly on `.stripped.elf` TA files and there is a slight difference as how they are
+generated. The in-tree `.stripped.elf` TA files are generated during the OP-TEE OS compilation, while
```

Comment:
during the OP-TEE OS compilation -> compilation of the TA DEV KIT

Reply:
Are you sure `make ta_dev_kit` generates `.stripped.elf` files? I just ran the make command and no `.stripped.elf` files have been generated. Or by TA DEV KIT you meant something else?

Reply:
I was referring to this comment: https://github.com/OP-TEE/optee_os/pull/4614#issuecomment-842327673

Reply:
That's correct:
```
$ make -s clean
$ make -s -j10 ta_dev_kit
$ find out/ -name *.stripped.elf
out/arm-plat-vexpress/ta/trusted_keys/f04a0fe7-1f5d-4b9b-abf7-619b85b4ce8c.stripped.elf
out/arm-plat-vexpress/ta/avb/023f8f1a-292a-432b-8fc4-de8471358067.stripped.elf
out/arm-plat-vexpress/ta/pkcs11/fd02c9da-306c-48c7-a49c-bbd827ae86ee.stripped.elf
$ 
```

Reply:
I'll change the documentation, however am I doing something wrong as I have no `.stripped.elf` files generated? Note the version is 3.12.0.

```
alenik@nb152:~/projects/github/optee/optee_os$ git status
HEAD losgelöst von 3.12.0
nichts zu committen, Arbeitsverzeichnis unverändert
alenik@nb152:~/projects/github/optee/optee_os$ make -s clean
alenik@nb152:~/projects/github/optee/optee_os$ make -s -j4 ta_dev_kit
alenik@nb152:~/projects/github/optee/optee_os$ find out/ -name *.stripped.elf
```

Reply:
@wamserma The documentation has been updated.

Reply:
Well, the docs should match 3.13, as this is the current release. Should we put a note in the docs, that this was introduced with 3.13?

Reply:
> Well, the docs should match 3.13, as this is the current release. 

Agreed.

> Should we put a note in the docs, that this was introduced with 3.13?

No, we have tagged releases of the doc already, good enough I think.